diff --git a/node_modules/better-dns/.bun-tag-4790c3f19cee23a6 b/.bun-tag-4790c3f19cee23a6
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/better-dns/.bun-tag-4f57672f9fc211d0 b/.bun-tag-4f57672f9fc211d0
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/better-dns/.bun-tag-8d6da95d8207400a b/.bun-tag-8d6da95d8207400a
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/better-dns/.bun-tag-d431dbdf421bfba8 b/.bun-tag-d431dbdf421bfba8
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/better-dns/.bun-tag-e93e5766697800ec b/.bun-tag-e93e5766697800ec
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/cjs/server/server.js b/dist/cjs/server/server.js
index 98d5f26c5f97ebc2f8dc6feb5b295d1c73d44bb5..fa07203a858e0cdea3f11951f1b9004f5c7cc5da 100644
--- a/dist/cjs/server/server.js
+++ b/dist/cjs/server/server.js
@@ -102,13 +102,16 @@ exports.DNSServer = DNSServer;
         }
         static async handleAXFRRequest(rinfo, question, response, dnsRecordStore, send) {
             const slaveConfig = await dnsRecordStore.getSlaveSettings(RequestHandler.normalizeName(question.name));
+            console.log(`Received AXFR request for zone ${question.name} from ${rinfo.address}:${rinfo.port}`);
             if (!slaveConfig) {
+                console.log(`AXFR request for zone ${question.name} denied: zone is not configured as a slave zone. ${slaveConfig}`);
                 response.header.rcode = 0x05; // REFUSED
                 send(response, false);
                 return;
             }
             //@ts-ignore
             if (!slaveConfig.isSlaveAllowed(rinfo.address().address)) {
+                console.log(`AXFR request for zone ${question.name} denied: requesting IP ${rinfo.address().address} is not allowed.`);
                 response.header.rcode = 0x05; // REFUSED
                 send(response, false);
                 return;
