diff --git a/dist/cjs/server/server.js b/dist/cjs/server/server.js
index e9de51d7909c5a51837234ac4f9cc5a897bc12aa..e7009f6ca66c502146f1faf18dde396701e12fa4 100644
--- a/dist/cjs/server/server.js
+++ b/dist/cjs/server/server.js
@@ -101,13 +101,16 @@ exports.DNSServer = DNSServer;
             return true;
         }
         static async handleAXFRRequest(rinfo, question, response, dnsRecordStore, send) {
+            console.log(`Received AXFR request for zone ${question.name} from ${rinfo.address}:${rinfo.port}`);
             const slaveConfig = await dnsRecordStore.getSlaveSettings(RequestHandler.normalizeName(question.name));
             if (!slaveConfig) {
+                console.log(`AXFR request for zone ${question.name} denied: zone is not configured as a slave zone. ${slaveConfig}`);
                 response.header.rcode = 0x05; // REFUSED
                 send(response, false);
                 return;
             }
             if (!slaveConfig.isSlaveAllowed(rinfo.address)) {
+                console.log(`AXFR request for zone ${question.name} denied: zone is not configured as a slave zone. ${slaveConfig}`);
                 response.header.rcode = 0x05; // REFUSED
                 send(response, false);
                 return;
